frontend/src/pages/Facharbeit/FacharbeitApplicationForm.tsx

```diff
@@ -0,0 +1,159 @@
+  const [unterrichtendeLehrkraft, setUnterrichtendeLehrkraft] = useState();
+  const [topicTextArea, setTopicTextArea] = useState('');
+
+  const betreuendeLehrkraefteList = lehrkraefteMock;
+  const unterrichtendeLehrkraftList = lehrkraefteMock;
+  const subjectList = subjectMock;
+
+  const handleStudentNameInputChange = (event) => {
+    setStudentName(event.target.value);
+  };
+  const handleStudentClassInputChange = (event) => {
+    setStudentClass(event.target.value);
+  };
+  const handleSubjectInputChange = (event) => {
+    setChoosenSubject(event.target.value);
+  };
+  const handleBetreuendeLehrkraftInputChange = (event) => {
+    setBetreuendeLehrkraft(event.target.value);
+  };
+  const handleUnterrichtendeLehrkraftInputChange = (event) => {
+    setUnterrichtendeLehrkraft(event.target.value);
+  };
+  const handleTopicInputChange = (event) => {
+    setTopicTextArea(event.target.value);
+  };
+  const handleSubmitFacharbeit = () => {
+    const requestBody = JSON.stringify({
+      studentId: 0,
+      topic: topicTextArea,
+      subjectId: choosenSubject,
+      responsibleTeacher: betreuendeLehrkraft,
+      teachingTeacher: unterrichtendeLehrkraft,
+    });
+    fetch('placeholder/api/facharbeit/chooseTopic', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: requestBody,
+    });
+  };
+
+  const handleGeneratePDF = () => {
+    alert('pdf generiert...');
+    const pdf = {
+      studentName,
+      studentClass,
+
+    };
+    console.log(pdf.toString());
+  };
+  return (
+    <div className="pageRoot">
+      <div style={{ width: '94vw' }}>
+        <h1>Facharbeit beantragen</h1>
+      </div>
+      <Paper style={{ display: 'flex', width: '90vw', padding: '2vw' }}>
+        <div
+          id="studentAndTopicInputs"
+          style={{
+            display: 'flex', flexDirection: 'column', width: '60%', marginRight: '3vw',
+          }}
+        >
+          <div style={{ display: 'flex', flexDirection: 'column', width: '100%' }}>
+            <h2>Schüler:in</h2>
+            <div style={{ display: 'flex', justifyContent: 'space-between', width: '100%' }}>
+              <TextField label="Name" variant="outlined" style={{ width: '50%' }} onChange={handleStudentNameInputChange} />
+              <TextField label="Klasse" variant="outlined" style={{ width: '30%' }} onChange={handleStudentClassInputChange} />
+            </div>
+          </div>
+          <TextField
+            label="Thema"
+            variant="outlined"
+            style={{ width: '100%', marginTop: '3vh' }}
+            multiline
+            rows={10}
+            onChange={handleTopicInputChange}
+          />
+        </div>
+        <div id="teacherInputsAndButtons" style={{ display: 'flex', flexDirection: 'column', width: '40%' }}>
+          <h2> </h2>
+          <FormControl style={{ width: '100%', marginTop: '32px' }} variant="outlined">
```

Comment:
hier hast du die Input Felder in so ein FormControl gepackt. Die Felder für das Thema oder den Namen allerdings nicht. Hat das einen bestimmten Grund?

Reply:
Dropdown Felder brauchen ein Wrapper um props wie "variant="outlined"" zu übergeben, die für die komponente als ganzes gelten. Bei TextInput kannst du sowas direkt an die TextInput komponente geben